.github/workflows/ci.yml

```diff
@@ -41,21 +41,21 @@ jobs:
         run: |
           sudo -E PATH=$PATH -s cargo test --all --features default
 
-      - name: Run cargo test - no-default-features
+      - name: Run cargo test - no-default-features,grpc
```

Comment:
This group of tests are trying to test each features individually, so can we avoid add feature grpc to each test case here?

Reply:
As the features `grpc`, `ttrpc` and `native-aa` are exclusive, we should specify one